plugins/chunter-resources/src/components/DirectMessageButton.svelte

```diff
@@ -20,8 +20,13 @@
   import chunter from '../plugin'
   import { createDirect } from '../utils'
   import { openChannelInSidebar } from '../navigation'
+  import { Asset } from '@hcengineering/platform'
 
   export let employee: Employee
+  export let width: string | undefined = undefined
```

Comment:
Is it used? I don't see where it is provided

Reply:
Sure, it's unused after redesign
I've removed these changes